manifests/fpm.pp

```diff
@@ -64,6 +64,7 @@
   Hash $pools                   = $php::real_fpm_pools,
   $log_owner                    = $php::log_owner,
   $log_group                    = $php::log_group,
+  Boolean $disable_configtest   = false,
```

Comment:
This would need a spec test to show that `validate_cmd` is as expected.

Reply:
Yes, indeed. I'll add a spec test for this parameter.